### PR TITLE
Fix tests for oracle extraction

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Fixes tests for oracle extraction performed in #1441 and removes some references.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

**Comment by Adamantios** [comment]
@dvilelaf @jmoreira-valory there are still some references, for example on [application_deployment](https://github.com/valory-xyz/open-autonomy/blob/f225f937863f9fc2821d663ecb33d90c4edc0c38/docs/application_deployment.md).
Could you take a look in case they should be updated?

File: packages/valory/agents/simple_abci/aea-config.yaml [hunk 1]
```diff
@@ -98,14 +98,14 @@ type: skill
...
   benchmark_tool:
     args:
-      log_dir: ${SKILL_ORACLE_ABCI_MODELS_BENCHMARK_TOOL_ARGS_LOG_DIR:str:/benchmarks}
+      log_dir: ${SKILL_SIMPLE_ABCI_MODELS_BENCHMARK_TOOL_ARGS_LOG_DIR:str:/benchmarks}
   params:
     args:
       consensus:
         max_participants: ${MAX_PARTICIPANTS:int:4}
-      round_timeout_seconds: ${ORACLE_PARAMS_ROUND_TIMEOUT_SECONDS:float:7.0}
+      round_timeout_seconds: ${SIMPLE_ABCI_PARAMS_ROUND_TIMEOUT_SECONDS:float:7.0}
```
Comment (DavidMinarsch) [inline packages/valory/agents/simple_abci/aea-config.yaml:106]:
```suggestion
      round_timeout_seconds: ${SIMPLE_ABCI_MODELS_PARAMS_ROUND_TIMEOUT_SECONDS:float:7.0}
```

File: tests/test_autonomy/test_replay.py [hunk 2]
```diff
@@ -45,28 +45,15 @@
...
     "environment": [
         "LOG_FILE=/logs/aea_0.txt",
         "ID=0",
-        "VALORY_APPLICATION=valory/oracle:0.1.0:QmUTuBXUbgBjBbiLR3kmBxZXQowseu2mc6xR5MVuX8UyXJ",
+        "VALORY_APPLICATION=valory/hello_world:0.1.0:bafybeideb6b5k4i6z7bm3p53eydxgknmwdefo2oshcnlxthjc6oxeox7ua",
         "ABCI_HOST=abci0",
         "MAX_PARTICIPANTS=4",
         "TENDERMINT_URL=http://node0:26657",
         "TENDERMINT_COM_URL=http://node0:8080",
-        "SKILL_ORACLE_ABCI_MODELS_PRICE_API_ARGS_URL=https://api.coingecko.com/api/v3/simple/price",
```
Comment (DavidMinarsch) [inline tests/test_autonomy/test_replay.py:57]: Is it safe to remove this for the test to retain meaning?

File: tests/test_docs/test_code_blocks.py [hunk 3]
```diff
@@ -137,19 +137,23 @@ class TestYamlSnippets(BaseTestDocCode):
...
         "docs/demos/simple_abci.md": {
             "code_files": ["packages/valory/skills/simple_abci/fsm_specification.yaml"],
         },
         "docs/networks.md": {"skip_blocks": [0]},
     }
 
+    skipped_files = [
+        "packages/valory/skills/oracle_deployment_abci/fsm_specification.yaml",
+        "packages/valory/skills/price_estimation_abci/fsm_specification.yaml",
+        "packages/valory/skills/oracle_abci/fsm_specification.yaml",
```
Comment (DavidMinarsch) [inline tests/test_docs/test_code_blocks.py:154]: They should no longer be part of the repo. Why skip?

File: tests/test_autonomy/test_replay.py (hunk 2 above)
Comment (Adamantios) [reply tests/test_autonomy/test_replay.py:57]: I believe yes, but we would need @angrybayblade's input too.

File: tests/test_docs/test_code_blocks.py (hunk 3 above)
Comment (Adamantios) [reply tests/test_docs/test_code_blocks.py:154]: :facepalm:

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**